<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="act_new"/>
    <addaction name="act_open"/>
    <addaction name="act_save"/>
   </widget>
   <widget class="QMenu" name="menubianyi_E">
    <property name="title">
     <string>编译(&amp;E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_B">
    <property name="title">
     <string>构建(&amp;B)</string>
    </property>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menubianyi_E"/>
   <addaction name="menu_B"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_new">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="act_open">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="act_save">
   <property name="text">
    <string>save(&amp;S)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
